External/TestDataSet/studyPar.json

```diff
@@ -0,0 +1,11 @@
+{"DatasetType":"raw",
```

Comment:
I believe this is called `data_description.json` in BIDS. Check out this nice format:
https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#source-vs-raw-vs-derived-data

Reply:
Oh, I thought with v1.7.0 we would just add a `rawdata` version and keep it like it is. You mean something like "DatasetType": "derived"?

Reply:
Not sure if I understand - Henk means that this whole file is called `data_description.json` and not `studyPar.json`.

Reply:
Yeah... but that's wrong, right? The dataset_description.json is in the rawdata folder. The studyPar.json is in the root directory. At least in the default case. And those are different files. Should I just add the rawdata version?